### ISSUE TITLE
Divisions at compile-time with results > 1 throw a floating point exception at compile-time

### ISSUE THREAD

**Issue by tkurby** [odin-lang/Odin#248]
Code in which a literal is used in an arithemtic expression at compile-time result in a floating-point exception.

Example:
```
x :: 4 / 2; // ERROR
```

The same happens when results of compile-time functions are used in an arithmetic expression.

Example:
```
x :: size_of(u32) / size_of(u16); // ERROR
```

When the result of the arithmetic expression is <= 1, no error occurs.

Example:
```
// all following expressions are OK

a :: size_of(u32) / 4;
b :: size_of(u32) / 8;
c :: 2 / size_of(u32);

x :: 4 / 4;
y :: 3 / 4;
z :: 2 / 4;

```

The same rules also apply when declaring an array with a static size.

Example:
```
main :: proc() {
    a : [4 / 2]int; // ERROR
    b : [2 / 4]int; // OK
} 
```

**Comment by gingerBill** [comment]
What operating system are you on?

**Comment by tkurby** [comment]
I am on Linux, Fedora 28 to be precise.

**Comment by gingerBill** [comment]
I cannot replicate this issue on Windows so I am guessing it is strictly a Linux issue.

**Comment by lunaticLipid** [comment]
@gingerBill 
I ran a debugger and it landed in `Odin/src/big_int.cpp:768` with `vn1 == 0`. At the start of the function you have
```
#if defined(GB_COMPILER_MSVC) && defined(GB_ARCH_64_BIT)
	*q = unsafe_udiv128(u1, u0, y, r);
#else
```
So if you compile with MSVC you don't run that code. See if you can reproduce it if you disable that #if or compile with something else.

**Comment by Breush** [comment]
@tkurby @lunaticLipid `vn1` being 0 on Linux have been fixed with 9623e5e032b078d546fee5e8ee9b23e9107a15eb. Does the issue still appear using the new master?

**Comment by tkurby** [comment]
I just tried it with the current master. Could not reproduce it, so it seems to be fixed.
